pom.xml

```diff
@@ -2116,6 +2116,7 @@
     <module>kie-pmml-bom</module>
     <module>narayana-integration-bom</module>
     <module>kie-platform-bom</module>
+    <module>kie-pmml-dependencies</module>
```

Comment:
It is not clear from the name of the module whether we want to import old or new PMML implementation. I know that we do not add word trusty to submodules. But this is a module which is expected to be imported by end user. I would suggest to use "kie-pmml-trusty-dependencies".